### PR TITLE
Ruby-Plugin-upgrade changes

### PR DESCRIPTION
**Pull request by prashelke**

### CODE REVIEW

File: .vs/VSWorkspaceState.json [hunk 1]
```diff
@@ -2,5 +2,6 @@
   "ExpandedNodes": [
     ""
   ],
+  "SelectedNode": "\\GingerRubyPlugin.sln",
```
Comment (Maheshkale447) [inline .vs/VSWorkspaceState.json]: remove unwanted files

File: GingerRubyPluginConsole/Ginger.PluginPackage.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "Id": "Ruby",
-  "Version": "1.0.0",
+  "Version": "2.0.1",
```
Comment (Maheshkale447) [inline GingerRubyPluginConsole/Ginger.PluginPackage.json]: change it to v2.0.0

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]

File: GingerRubyPluginConsole/Ginger.PluginPackage.json (hunk 2 above)
Comment (prashelke) [reply GingerRubyPluginConsole/Ginger.PluginPackage.json]: Done

File: .vs/VSWorkspaceState.json (hunk 1 above)
Comment (prashelke) [reply .vs/VSWorkspaceState.json]: done

**Pull request closed without merging**